lib/config/index.js

```diff
@@ -59,6 +57,8 @@ async function parseConfigs(env, argv) {
   // Set log level
   logger.levels('stdout', config.logLevel);
 
+  logger.debug('Parsing configs');
```

Comment:
No, let's revert this. If the user has set `LOG_LEVEL=debug` in env then the message will be logged where it was, and it's good to log "Parsing configs" before the parsing is done.

Reply:
Oh, yes indeed. Though it is misleading when you use CLI `--log-level debug` and this log message does not appear.

Reply:
It's a little chicken and egg but basically if you need logs of the parsing itself then you need to do it via env, as we have a special rule for that.